packages/preact-iso/router.d.ts

```diff
@@ -8,7 +8,7 @@ interface LocationHook {
 	url: string;
 	path: string;
 	query: Record<string, string>;
-	route: (url: string) => void;
+	route: (url: string | { url: string, replace?: boolean }) => void;
```

Comment:
Wondering if we can use `(url, replace)` here? That would match `preact-router`.